src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -143,6 +143,7 @@ export default class FilingHistoryList extends Mixins(FilingMixin) {
   is (filing: ApiFilingIF): string {
     switch (true) {
       case filing.availableOnPaperOnly: return 'paper-filing' // must come first
+      case EnumUtilities.isTypeAdminFreeze(filing): return null
```

Comment:
I never tried this, but looking at the template code, an expansion panel is probably still created for this item.

Also this might throw off the count of items in this list.

It may be better to simply filter out the "hidden" filings in filingHistoryStore.ts::getFilings() but I haven't tried that either.

Reply:
Code was reverted. Resolving.